config.template/faf-clans/faf-clans.env

```diff
@@ -1,7 +1,3 @@
-API_BASE_URL=https://api.dev.faforever.com
-OAUTH_CLIENT_ID=83891c0c-feab-42e1-9ca7-515f94f808ef
-OAUTH_REDIRECT_URI=https://clans.faforever.com
-
-#VIRTUAL_HOST=clans.faforever.com
-#LETSENCRYPT_HOST=clans.faforever.com
-#LETSENCRYPT_EMAIL=admin@faforever.com
+API_BASE_URL=https://faf-java-api:8010
```

Comment:
If it's the browser that connects to this URL, then it has to be `http://localhost`